dbfit-java/core/src/main/java/dbfit/api/AbstractDbEnvironment.java

```diff
@@ -240,8 +240,8 @@ public String buildInsertCommand(String tableName,
         return sb.toString();
     }
 
-    public DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors) {
-        return new DbStoredProcedureCall(this, name, accessors);
+    public DbStoredProcedure newDbStoredProcedure(String name) {
```

Comment:
> Why do you think we would we pass in accessors to the DbStoredProcedure constructor? It loads it's own accessors from database metadata.

If there are multiple procedures with same name - how would it know which one to pick from the database metadata? We don't support that yet - it's just a hint that in future we might need something more than the name. (To be honest - perhaps it's better to create a domain class instead of passing plain string and an array... So that part is anyway a candidate for change in future)